components/user/UserTabs.js

```diff
@@ -92,7 +92,7 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           id="tabs"
           name="tabs"
           onChange={changeTab}
-          className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
+          className="block w-full rounded-md -mb-px border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
```

Comment:
I don't think this is required?

Reply:
-mb-pr is the part of original code base. At first while resolving this issue, I removed it but then you told me not to move user tabs down. So added it back to the original code base.

Reply:
I don't think it is, otherwise it would not be shown as being added?

You can check in `main` https://github.com/EddieHubCommunity/LinkFree/blob/main/components/user/UserTabs.js#L95

Reply:
okay maybe my codebase was not updated or something of that sort. I will do it asap.